proteus/TwoPhaseFlow/TwoPhaseFlowProblem.py

```diff
@@ -120,12 +129,14 @@ def __init__(self,
                  final_time,
                  dt_init=0.001,
                  dt_output=None,
-                 nDTout=None):
+                 nDTout=None,
+                 dt_fixed=None):
```

Comment:
Can you remind me what is `dt_fixed` for? I think it has the same use to what I call `dt_output`; basically how often in real time units you want to output the solution. Is this what `dt_fixed` is for?

Reply:
`dt_fixed` is the maximum time step to be taken, but not for archiving (which I think is what `dt_output` was here, but correct me if I am wrong) and can be used with CFL. At each time step, `dt` will be whatever is smaller between the time step respecting the CFL condition and `dt_fixed`

Reply:
Understood and yes, `dt_output` is for archiving.

Reply:
Btw, is this way of controlling the outputting convenient for you guys?

Reply:
I think it is fine, we used `nsave` before, which is the number of save per second, but using the rate of save works too